test/evergreen.yml

```diff
@@ -1604,12 +1614,13 @@ buildvariants:
   run_on:
   - ubuntu1804-test
   expansions:
-    test_env_vars: PATH=/opt/mongodbtoolchain/v3/bin:$PATH LD_LIBRARY_PATH=$(pwd)/.libs top_srcdir=$(pwd)/.. top_builddir=$(pwd)
+    test_env_vars: PATH=/opt/mongodbtoolchain/v3/bin:$PATH LD_LIBRARY_PATH=$(pwd)/.libs top_srcdir=$(pwd)/.. top_builddir=$(pwd) CFLAGS="-fPIC -ggdb"
```

Comment:
Should the addition of CFLAGS setting be put as part of `posix_configure_flags`? 
Same comment for other POSIX build variants below.